src/Sylius/Bundle/CoreBundle/Command/InstallCommand.php

```diff
@@ -67,7 +67,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $outputStyle->writeln('<info>Installing Sylius...</info>');
         $outputStyle->writeln($this->getSyliusLogo());
 
-        $this->ensureDirectoryExistsAndIsWritable((string) $this->getContainer()->getParameter('kernel.cache_dir'), $output);
+        $this->ensureDirectoryExistsAndIsWritable((string) $this->container->getParameter('kernel.cache_dir'), $output);
```

Comment:
Looking at the tests, it looks like it's not working after the refactor, will you try to fix the problem?